css/index.css

```diff
@@ -143,4 +414,90 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
+}
+
+
+@media all and (max-width: 600px) {
+
```

Comment:
Great that your are comfortable with '@media'! This should make your week much easier!